docker-dev-setup/README.md

```diff
@@ -77,6 +80,8 @@ docker exec -it mars_mysql mysql --user=mars --password=password
 
 Alternatively, you can access the database directly from you host machine (via an application of your choice) at this address: `127.0.0.1:3307`
 
+You can reset the database by running `docker exec -it mars_dev bash -c "php artisan migrate:fresh --seed"`.
```

Comment:
Isn't there a way to open docker's terminal? (at least PHPStorm should have this feature if you are using that). so that we can just forward them to the same "everyday use" part.

(also applies to the docker command for accessing the db and below to generate IDE helper files)